rpi3/firmware/uboot.env.tpl

```diff
@@ -0,0 +1,70 @@
+arch=arm
+atf_load_addr=0x08400000
+baudrate=115200
+board=rpi
+board_name=3 Model B
+board_rev=0x8
+board_rev_scheme=1
+board_revision=0xA02082
+boot_a_script=load ${devtype} ${devnum}:${distro_bootpart} ${scriptaddr} ${prefix}${script}; source ${scriptaddr}
+boot_efi_binary=load ${devtype} ${devnum}:${distro_bootpart} ${kernel_addr_r} efi/boot/bootaa64.efi; bootefi ${kernel_addr_r}
+boot_extlinux=sysboot ${devtype} ${devnum}:${distro_bootpart} any ${scriptaddr} ${prefix}extlinux/extlinux.conf
+boot_it=booti ${kernel_addr_r} - ${fdt_addr_r}
+boot_net_usb_start=usb start
+boot_prefixes=/ /boot/
+boot_script_dhcp=boot.scr.uimg
+boot_scripts=boot.scr.uimg boot.scr
+boot_targets=mmc0 usb0 pxe dhcp
+bootargs=console=ttyS0,115200 root=/dev/mmcblk0p2 rw rootfs=ext4 ignore_loglevel dma.dmachans=0x7f35 rootwait 8250.nr_uarts=1 elevator=deadline fsck.repair=yes smsc95xx.macaddr=@MAC@ bcm2708_fb.fbwidth=1920 bcm2708_fb.fbheight=1080 vc_mem.mem_base=0x3dc00000 vc_mem.mem_size=0x3f000000
+bootcmd=run load_kernel ; run load_dtb; run load_firmware; run set_bootargs; run boot_it
```

Comment:
So this will be replace by that, right?
https://github.com/OP-TEE/build/blob/master/rpi3/firmware/uboot.env.txt#L7